.github/workflows/build-and-push-image-development.yml

```diff
@@ -59,3 +59,26 @@ jobs:
               --resource-group ${{ secrets.DEVELOPMENT_ACA_RESOURCE_GROUP }} \
               --image ${{ secrets.DEVELOPMENT_ACR_URL }}/aca-app:${{ steps.prepare-tags.outputs.deploy-version }} \
               --output none
+
+  cypress-tests:
+    needs: build-and-push-image-development
+    environment: development
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: CypressTests
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Npm install
+        run: npm install
+      - name: Run cypress
+        run: npm run cy:run
```

Comment:
@dangood84 Are there any command line flags we need to be passing to cypress here? e.g. api keys